src/lmic/lmic_config_preconditions.h

```diff
@@ -64,6 +64,7 @@ Revision history:
 // constants for comparison
 #define LMIC_REGION_eu868    1
 #define LMIC_REGION_us915    2
+#define LMIC_REGION_au915    2
```

Comment:
This should be a unique value. By convention, these match the subsections in the LoRaWAN regional spec.I've not had a chance to review the updates that enable AU915. Because the regulatory regime is different, even if this is identical to US915, we need to think about keeping it separate here. (There's no necessary connection, it's just an interesting fact.)

Reply:
Its the same band per se, the US ISM actually starts lower.
so we are not able to use the first sub band,

One other difference is the use of EIRP here.

Reply:
@Scobber What document defines AU915 (as opposed to "AU915-928", which is defined by section 2.6 of the LoRaWAN regional parameters document)?

This is important because we (architecturally) must use a new value for `LMIC_REGION_au915` -- we can't use `2` because it's already in use. I just checked the latest regional specs, and this doesn't seem to be published. Is this something the LoRa Alliance defined, or something someone else defined? Thanks!